charts/prometheus-systemd-exporter/values.yaml

```diff
@@ -51,7 +51,8 @@ env: {}
 ##    VARIABLE: value
 
 # extraArgs allows to pass command line arguments, as described on https://github.com/prometheus-community/systemd_exporter?tab=readme-ov-file#configuration
-extraArgs: []
+extraArgs:
+  - --systemd.collector.unit-include=kubelet.service|docker.service
```

Comment:
if people have already set extraArgs it will be breaking since it will disable it, prefer to use a new field:
```yaml
config:
  log:
    level: info
  systemd:
    collector:
      unit-include:
        - kubelet.service
        - docker.service
```

Reply:
My bad! That was my original approach but then thought `extraArgs` would be simpler to implement (didn't realize it was breaking for those already using it).

Will refactor it